chromadb/test/property/test_filtering.py

```diff
@@ -305,3 +305,22 @@ def test_boolean_metadata(api: ServerAPI) -> None:
     res = coll.get(where={"test": True})
 
     assert res["ids"] == ["1", "3"]
+
+
+def test_get_empty(api: ServerAPI) -> None:
```

Comment:
Looks good just wondering if we want to also add the where. It is essentially the same thing just a different use case.

Reply:
added